code-review-trainer-app/src/components/CodeReviewPractice.tsx

```diff
@@ -178,14 +215,22 @@ const CodeReviewPractice = () => {
           <div className="submit-section">
             <button
               onClick={handleSubmitReview}
-              disabled={!reviewComments.trim()}
+              disabled={!reviewComments.trim() || isSubmitting}
               className={`submit-button ${
-                reviewComments.trim() ? "enabled" : "disabled"
+                reviewComments.trim() && !isSubmitting ? "enabled" : "disabled"
               }`}
             >
-              Submit Review
+              {isSubmitting ? "Submitting..." : "Submit Review"}
```

Comment:
Use Ellipsis in place of "..."

also, can you add this to your instructions file?